migration/rollout_new_bvls_service.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# The purpose of this script is to ease the rollout of the new BVLS service by automating the toggling of features
+# by applying changes to feature toggles set up as K8's secrets and restarting the pods in the affected namespaces.
+
+read -r -p "Enter the environment you wish to rollout the new BVLS service - dev/preprod/prod: " ENVIRONMENT
+
+if [ -z "$ENVIRONMENT" ]
+then
+  echo "No environment specified."
+  exit 99
+fi
+
+while true; do
+
+read -r -p "You are about to rollout the new BVLS service in the '$ENVIRONMENT' environment. Are you sure you want to proceed? (y/n) " yn
+
+case $yn in
+	[yY] ) echo ok, proceeding...;
+		break;;
+	[nN] ) echo exiting...;
+		exit;;
+	* ) echo invalid response;;
+esac
+
+done
+
+# Each service that needs changes applying should be added as a separate file and sourced below.
+source ./whereabouts-api.sh "$ENVIRONMENT"
```

Comment:
Plan here is we can just add the other namespaces as separate files.  Just need to pass the `$ENVIRONMENT` through.